codespell_lib/data/dictionary.txt

```diff
@@ -297,6 +297,7 @@ accessiblilty->accessibility
 accessile->accessible
 accessintg->accessing
 accessisble->accessible
+accessoires->accessories
```

Comment:
In french and german accessoire is the word for accessory. Maybe it's worth to add it?
grep.app (almost) only finds, french, german, dutch and translations.

Reply:
Added

Reply:
Hmmm, I'd kinda been assuming that `dictionary.txt` only had english-language translations? It feels a bit "all bets are off" if we're also going to be including possible-typo-corrections for other languages too?

I don't mean to sound like an arrogant English-speaker (ich auch spreche nur ein bisschen Deutsch), but IMHO if we want to do native-language spellchecking then it would make sense to have e.g. `dictionary_de-DE.txt` etc. ?

Reply:
Oh! Having just looked at the actual commits, I see I misunderstood, sorry! :disappointed:

Reply:
Although there is an argument for potentially moving words that are valid in other languages out. With a mixed language codebase you spend a lot of time skipping stuff. I'm not sure how you'd easily handle words being valid in multiple languages though.